stdpopsim/catalog/ApiMel/genome_data.py

```diff
@@ -3,22 +3,24 @@
     "assembly_accession": "GCA_003254395.2",
     "assembly_name": "Amel_HAv3.1",
     "chromosomes": {
-        "CM009931.2": {"length": 27754200, "synonyms": ["NC_037638.1"]},
-        "CM009932.2": {"length": 16089512, "synonyms": ["NC_037639.1"]},
-        "CM009933.2": {"length": 13619445, "synonyms": ["NC_037640.1"]},
-        "CM009934.2": {"length": 13404451, "synonyms": ["NC_037641.1"]},
-        "CM009935.2": {"length": 13896941, "synonyms": ["NC_037642.1"]},
-        "CM009936.2": {"length": 17789102, "synonyms": ["NC_037643.1"]},
-        "CM009937.2": {"length": 14198698, "synonyms": ["NC_037644.1"]},
-        "CM009938.2": {"length": 12717210, "synonyms": ["NC_037645.1"]},
-        "CM009939.2": {"length": 12354651, "synonyms": ["NC_037646.1"]},
-        "CM009940.2": {"length": 12360052, "synonyms": ["NC_037647.1"]},
-        "CM009941.2": {"length": 16352600, "synonyms": ["NC_037648.1"]},
-        "CM009942.2": {"length": 11514234, "synonyms": ["NC_037649.1"]},
-        "CM009943.2": {"length": 11279722, "synonyms": ["NC_037650.1"]},
-        "CM009944.2": {"length": 10670842, "synonyms": ["NC_037651.1"]},
-        "CM009945.2": {"length": 9534514, "synonyms": ["NC_037652.1"]},
-        "CM009946.2": {"length": 7238532, "synonyms": ["NC_037653.1"]},
-        "CM009947.2": {"length": 16343, "synonyms": ["NC_001566.1", "MT"]},
```

Comment:
Seems a mistake to get rid of the `MT` synonym?

Reply:
But these are hella weird chromosome names anyhow, so meh?

Reply:
so the way we have things set up, synonyms are automatically generated by what goes in from ensembl. they must have removed it